Source/Shaders/BillboardCollectionFS.glsl

```diff
@@ -15,13 +15,28 @@ void main()
 #else
     vec4 vertexColor = v_color;
 #endif
-    
+
     vec4 color = texture2D(u_atlas, v_textureCoordinates) * vertexColor;
-    if (color.a == 0.0)
+
+#ifdef RENDER_FOR_PICK
```

Comment:
Can you add a comment here and to the point shader that explains this?  For example: fully transparent parts are not pickable, and the billboard/point is rendered twice: the opaque pass discards translucent fragments, and the translucent pass discards opaque ones.